src/python/pants/backend/python/goals/lockfile.py

```diff
@@ -77,7 +77,7 @@ def register_options(cls, register) -> None:
                 "Only generate lockfiles for the specified resolve(s).\n\n"
                 "Resolves are the logical names for the different lockfiles used in your project. "
                 "For your own code's dependencies, these come from the option "
-                "`[python].experimental_resolves_to_lockfiles`. For tool lockfiles, resolve "
+                "`[python].experimental_resolves`. For tool lockfiles, resolve "
```

Comment:
Consider dropping `experimental` here, since we're planning to stabilize this in 2.10.x?

Reply:
We are aiming to, but I think we should probably wait. Too many unknowns like that we need https://github.com/pantsbuild/pex/issues/1402.